.github/workflows/cd-pypi.yml

```
@@ -17,7 +17,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install .[train,dev]
+        pip install .[train,rllib,torch,dev]
```

In alphabetical order.

```suggestion
        pip install .[dev,rllib,torch,train]
```